src/qilib/configuration_helper/instrument_adapter.py

```diff
@@ -99,12 +102,40 @@ def accept(self, visitor: Visitor) -> None:
 
         Args:
             visitor: An implementation of the Visitor interface.
-
         """
         visitor.visit(self)
 
+    def __notify_and_remove_none_values(self, snapshot: Dict[str, Any]) -> Dict[str, Any]:
+        """ Gets all setted parameters from the QCoDeS snapshot.
```

Comment:
```suggestion
        """ Return parameters from the QCoDeS snapshot which are not None
```
A Parameter can also be not None if no set occurred (but just a get).